<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>libraryManageUser</class>
 <widget class="QWidget" name="libraryManageUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{background-color: rgb(225, 225, 225);border:2px groove gray;border-radius:15
px;padding:2px 4px;border-style: outset;}&quot;
                                           &quot;QPushButton:hover{background-color:rgb(229, 241, 251); color: black;}&quot;
                                           &quot;QPushButton:pressed{background-color:rgb(204, 228, 247);border-style: inset;}
</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 14px ;  
}
 
QLineEdit:hover {
    border: 2px solid #8fb2c9;  
}
</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>531</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListView {
	font: 25 9pt &quot;Microsoft YaHei&quot;;
	border: 15px solid #e2e1e4; /* 设置边框的大小，样式，颜色 */
	border-radius: 10px;
    background-color: #e2e1e4
}
QListView::item:hover {
	background-color: transparent;
	padding: 10px;
	border-left: 3px solid rgb(130, 130, 130);
}
QListView::item:selected {
	background-color: transparent;
	color: black;
	padding: 10px;
	border-left: 3px solid black;
}
QListView::item {
    height: 60px;
    background-color: #e2e1e4
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>400</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(225, 225, 225);border:2px groove gray;border-radius:13px;padding:2px 4px;border-style: outset;}
                                           QPushButton:hover{background-color:rgb(229, 241, 251); color: black;}
                                           QPushButton:pressed{background-color:rgb(204, 228, 247);border-style: inset;}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_help">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(225, 225, 225);border:2px groove gray;border-radius:13px;padding:2px 4px;border-style: outset;}
                                           QPushButton:hover{background-color:rgb(229, 241, 251); color: black;}
                                           QPushButton:pressed{background-color:rgb(204, 228, 247);border-style: inset;}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(225, 225, 225);border:2px groove gray;border-radius:13px;padding:2px 4px;border-style: outset;}
                                           QPushButton:hover{background-color:rgb(229, 241, 251); color: black;}
                                           QPushButton:pressed{background-color:rgb(204, 228, 247);border-style: inset;}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_numOfUser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
